### PR TITLE
Rowan/feature/bring new gs

### PR DESCRIPTION
**Pull request by rrasmuss4200**

### CODE REVIEW

**Comment by rrasmuss4200** [comment]
The same as my other pr but with the udated gs. Shoudla made it on the same branch lol

File: ex3_ground_station/.gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "ex3_shared_libs"]
```
Comment (dheadrick1618) [inline ex3_ground_station/.gitmodules:1]: Is this still needed, if we are not using submodules anymore?

File: ex3_ground_station/README.md [hunk 2]
```diff
@@ -1,51 +1,55 @@
...
 
-Ground station software to interface with Ex Alta 3
+## Overview
 
-This (very) preliminary strawman uses Rocket to implement a simple WebServer
-that can send messages to the OBC prototype in ex3_obc_fe2o3.
+The EX3 Ground Station software interfaces with the Ex Alta 3 satellite, offering command and control capabilities through a web-based dashboard and a command-line interface. The project is organized as a Rust workspace with multiple components including a backend server, a web dashboard, migration tools, and a CLI for direct OBC communication.
 
-To build/run:
+## Workspace Structure
```
Comment (dheadrick1618) [inline ex3_ground_station/README.md:7]: Great job!

**Review by dheadrick1618 — CHANGES_REQUESTED** [review]
LGTM once the submodule reference is removed, and a main README is added to the root dir of this repo explaining the contents, and briefly why we put everything from GS and FSW into the same repo.

**Review by dheadrick1618 — APPROVED** [review]
LGTM. Previous command uplink tall thin demo works. Good job :)

**Pull request merged**